iOS/Shimmer/source/Shimmer/ApiDefinition.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+
+using UIKit;
+using Foundation;
+using ObjCRuntime;
+using CoreGraphics;
+
+namespace Shimmer.iOS
+{
+    // @protocol FBShimmering <NSObject>
+    [Protocol, Model]
+    [BaseType(typeof(NSObject))]
+    interface FBShimmering
```

Comment:
I recommend removing the `FB` prefix. Also, you can add the `IFBShimmering` due `FBShimmering` is a Protocol.

Reply:
According to Objective-C headers, there are two classes missing: `FBShimmeringLayer` and `FBShimmeringView`